app/views/_pod-template-container.html

```diff
@@ -8,8 +8,10 @@
  -->
 
 <div class="pod-template">
-  <div class="component-label"><span ng-bind-template="{{labelPrefix||'Container'}}:"></span> {{container.name}}</div>
-
+  <div ng-if="!detailed">
```

Comment:
Could keep the markup simpler by applying the ng-if to div.component-label.section-label rathern than adding a new div.